specification/network/resource-manager/Microsoft.Network/preview/2022-04-01-preview/networkManagerSecurityUserConfiguration.json

```diff
@@ -186,6 +186,15 @@
           "200": {
             "description": "OK"
           },
+          "202": {
```

Comment:
Security user does not support force delete.

Reply:
Discussed offline, will keep it, and implement it in backend